test/writing.jl

```diff
@@ -193,14 +193,14 @@ end
             command = VideoIO.FFMPEG.ffprobe,
             collect = true,
         )
-        @test parse(Float64, measured_dur_str[1]) == target_dur
+        @test parse(Float64, measured_dur_str[1]) ≈ target_dur
     end
 end
 
 @testset "Encoding video with float frame rates" begin
     n = 100
     fr = 29.5 # 59 // 2
-    target_dur = 3.39
+    target_dur = 3.389831
```

Comment:
Should this be n / fr, or there's a good reason to spell the number out explicitly?

Reply:
```suggestion
    target_dur = n / fr
```